civis/polling.py

```diff
@@ -104,6 +117,64 @@ def __repr__(self):
                                             self._civis_state)
             return out
 
+    def _subscribe(self):
+        pubnub = None
+        if self.poller_args:
+            class JobCompleteListener(SubscribeCallback):
+                def __init__(self, job_id, callback_function):
+                    self.job_id = job_id
+                    self.callback_function = callback_function
+
+                def message(self, pubnub, message):
+                    try:
+                        result = message.message
+                        if result['object']['id'] == self.job_id \
```

Comment:
Yeah. The messages have `run_id`. The blocker right now is from Platform because `POST /jobs/:id/runs` currently returns a 204 with an empty response. I have a PR open to return the run that is created, so once that is merged (probably early next week) I'll update this to use run and also change the polling to use the runs endpoint instead of jobs.